<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PositionResultView</class>
 <widget class="QWidget" name="PositionResultView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>663</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="lb_guid">
     <property name="text">
      <string>6B29FC40-CA47-1067-B31D-00DD010662DA</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>RPnL:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_pnl">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Open time:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_open_time">
       <property name="text">
        <string>MM.DD || HH:MM:SS</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Opened time (m):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lb_opened_time_m">
       <property name="text">
        <string>0.0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pb_chart">
     <property name="text">
      <string>Chart</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
